src/client-agent/main.c

```diff
@@ -56,6 +56,13 @@ int main(int argc, char **argv)
     const char *group = GROUPGLOBAL;
     const char *cfg = DEFAULTCPATH;
 
+    /* To check if is valid server address */
+    bool is_valid = false;
+    int i = 0;
+    int min_size = 1;
+    char val_default1[12] = "MANAGER_IP\0";
```

Comment:
String literals are always null terminated.

Should be discussed if this "constant literals" should be here or defined in a header file. In any case having them as variable is not needed.